src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java

```diff
@@ -66,6 +67,12 @@ public ErrorMessage handleRuntimeException(RuntimeException e, ServletWebRequest
         return logAndGetErrorMessage(request, e, Level.ERROR);
     }
 
+    @ExceptionHandler(InvalidLocationInitFileCreatingDataException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
We already have such response status with WARN level logging. You just need to update such part adding new exception and changing the method name:
```
    @ExceptionHandler(InvalidDtoException.class)
    @ResponseStatus(HttpStatus.BAD_REQUEST)
    public ErrorMessage handleInvalidDtoException(Exception e, ServletWebRequest request) {
        return logAndGetErrorMessage(request, e, Level.WARN);
    }
```